vtkext/private/module/vtkF3DMetaImporter.cxx

```diff
@@ -311,6 +311,116 @@ std::string vtkF3DMetaImporter::GetOutputsDescription()
   return description;
 }
 
+//----------------------------------------------------------------------------
+vtkF3DImporter::AnimationSupportLevel vtkF3DMetaImporter::GetAnimationSupportLevel()
+{
+#if VTK_VERSION_NUMBER < VTK_VERSION_CHECK(9, 4, 20250507)
+  vtkF3DImporter::AnimationSupportLevel levelAccum = vtkF3DImporter::AnimationSupportLevel::MULTI;
+#else
+  vtkImporter::AnimationSupportLevel levelAccum = vtkImporter::AnimationSupportLevel::NONE;
+  for (const auto& importerPair : this->Pimpl->Importers)
+  {
+    AnimationSupportLevel level = importerPair.Importer->GetAnimationSupportLevel();
+    switch (level)
+    {
+      case vtkImporter::AnimationSupportLevel::NONE:
+        /*
+        switch(levelAccum)
```

Comment:
The commented code is for review clarity but could be considered documentation, let me know what you think @meak

Reply:
Let's document it in plain english.